src/test/java/guitests/UndoCommandTest.java

```diff
@@ -92,6 +92,6 @@ private void assertUndoUsingAcceleratorSuccess(TestTaskList expectedList, String
     
     private void assertNoMoreUndos() {
         commandBox.runCommand("undo");        
-        assertResultMessage(UndoCommand.MESSAGE_NO_PREVIOUS_COMMANDS);
+        assertResultMessage(Messages.MESSAGE_NO_PREVIOUS_COMMANDS);
```

Comment:
The rest of the app imports the message like this:
import static seedu.taskitty.commons.core.Messages.MESSAGE_NO_PREVIOUS_COMMANDS;
and just uses MESSAGE_NO_PREVIOUS_COMMANDS (without the Messages.)
Should we stick with that way, or change everything to the way you are doing now?

Also, why is it in Messages class? Isn't it sort of a undo command only message?